dashboard/test/models/level_source_test.rb

```diff
@@ -39,31 +14,4 @@ class LevelSourceTest < ActiveSupport::TestCase
     refute level_source.valid?
     assert_equal ['Data is invalid'], level_source.errors.full_messages
   end
```

Comment:
Locally, this test takes over 40 seconds. @breville, we should investigate whether we can accomplish the same testing without calling `LevelSource.find_identical_or_create` (that itself uses the Rails cache).